manifest.json

```diff
@@ -0,0 +1,132 @@
+{
+    "app-id": "org.kiwix.Client",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "kiwix-desktop",
+    "rename-desktop-file": "kiwix-desktop.desktop",
+    "rename-icon": "kiwix-desktop",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-data"
```

Comment:
Does this actually work?

Reply:
Hum yes. Why it should not ?
Do I missed something ?

Reply:
I just don't understand it's supposed to achieve. XDG_DATA_DIR env var is overwritten by flatpak and application has access to it by default.

Reply:
The purpose was to allow the application to access to XDG_DATA_DIR.
But I've just test without it and it still works, good to know.
I will remove it.